ci/unreal-engine.version

```diff
@@ -1 +1,2 @@
+HEAD feature/4.25-support
```

Comment:
I remember something being related to ordering of these lines for EP building. I assume the top one is the one used for EP?

Reply:
Yeah, will need to be updated to 4.25-SpatialOSUnrealGDK anyway.